ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -3110,3 +3110,21 @@ literals:
 - ultraviolet_irradiance_sensor:
     fixed_min: 0.0
     flexible_max: 2000.0
+
+- floor_status:
+  - GROUND
+  - FIRST
+  - SECOND
+  - THIRD
+  - FOURTH
+  - FIFTH
+  - SIXTH
+
+- max_load_alarm:
+  - ACTIVE
+  - INACTIVE
+
+- elevator_status:
```

Comment:
Is this being used in any abstract or general types?

Reply:
fixed thank you

Reply:
@kevin-hereworks please clarify the use of this field. To me this is the category/space type of the elevator and is more like metadata than telemetry.